index.html

```diff
@@ -34,6 +34,80 @@
 
 <body class="bg-white">
 
+  <div class="bg-white border-b">
+
+    <div class="flex mb-4 ml-8 mr-8 mt-4">
+      <div class="flex w-1/2">
+        <h1 class="text-black text-center font-bold font-serif text-5xl text-left align-bottom">FrieschDagblad</h1>
+        <p class="text-blue-600 align-bottom mt-8 ml-2 font-bold">
+          <time datetime="2020-2-6">Dinsdag 2 mei 2020</time>
+        </p>
+      </div>
+
+      <div class="flex w-1/2 items-center">
+        <div class="lg:flex-grow text-right">
+          <a href="#responsive-header"
+            class="block lg:inline-block lg:mt-0 text-black hover:text-gray-600 hover:border-b-2 border-blue-600 mr-4 uppercase">
+            digitale krant
+          </a>
+          <a href="#responsive-header"
+            class="block lg:inline-block lg:mt-0 text-black hover:text-gray-600 hover:border-b-2 border-blue-600 mr-4 uppercase">
+            services
+          </a>
+          <a href="#responsive-header"
+            class="block lg:inline-block lg:mt-0 text-black hover:text-gray-600 hover:border-b-2 border-blue-600 mr-4 uppercase">
+            inloggen
+          </a>
+          <a href="#responsive-header"
+            class="block border border-blue-600 bg-blue-600 p-1 pr-2 pl-2 rounded lg:inline-block lg:mt-0 text-white hover:bg-blue-400 hover:border-blue-400 mr-4 uppercase">
+            abonneren
+          </a>
```

Comment:
Zou je de volgorde van de knoppen nog kunnen aanpassen en ervoor zorgen dat de abonneeknop een ronding krijgt net zoals in het design in Figma.